python/ray/tests/test_runtime_context.py

```diff
@@ -345,6 +345,16 @@ def foo(self):
     actor = FooActor.remote()
     ray.get(actor.foo.remote())
 
+    # actor name
+    @ray.remote
+    class NamedActor:
+        def name(self):
+            return ray.get_runtime_context().get_actor_name()
+
+    ACTOR_NAME = "actor_name"
+    named_actor = NamedActor.options(name=ACTOR_NAME).remote()
+    assert ray.get(named_actor.name.remote()) == ACTOR_NAME
+
```

Comment:
Can you also test the case where we don't specify `name=` and test what `get_actor_name()` returns.